hugegraph-llm/README.md

```diff
@@ -15,25 +15,28 @@ graph systems and large language models.
 2.  Use natural language to operate graph databases (Gremlin/Cypher)
 3.  Knowledge graph supplements answer context (GraphRAG)
 
-## Environment Requirements
+## 2 Environment Requirements
 
 - python 3.9+ 
 - hugegraph-server 1.2+
 
-## Preparation
+## 3 Preparation
+
+1. Start the HugeGraph database, you can run it via [Docker](https://hub.docker.com/r/hugegraph/hugegraph)/[Binary Package](https://hugegraph.apache.org/docs/download/download/).  
+    Refer to detailed [doc](https://hugegraph.apache.org/docs/quickstart/hugegraph-server/#31-use-docker-container-convenient-for-testdev) for more guidance
 
-1. Start the HugeGraph database, you can run it via [Docker](https://hub.docker.com/r/hugegraph/hugegraph)/[Binary Package](https://hugegraph.apache.org/docs/download/download/).
-Refer to detailed [doc](https://hugegraph.apache.org/docs/quickstart/hugegraph-server/#31-use-docker-container-convenient-for-testdev) for more guidance (PS: Graph visualization in step8)
 2. Clone this project
     ```bash
     git clone https://github.com/apache/incubator-hugegraph-ai.git
     ```
+    
```

Comment:
why need empty line for `1. 2. 3.`

Reply:
Previously, I thought I was looking at unrendered markdown text. After all, line spacing is comfortable🧐

But forgetting to take the serial number into account will result in the wrong order, now correct it immediately🫡